src/ServiceControl/Hosting/Commands/CustomLivetimes/PersisterInitializingWindowsServiceLifetime.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Hosting.WindowsServices;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using ServiceControl.Persistence;
+
+sealed class PersisterInitializingWindowsServiceLifetime : WindowsServiceLifetime, IHostLifetime
```

Comment:
Is this deliberately in the default namespace? Also it should be Lifetime instead of Livetimes in the folder

Reply:
Changed the folder to HostBuilderLifetimes but kept the namespace to `namespace Microsoft.Extensions.Hosting` as no need to have a namespace with only 2 types.